clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -566,7 +566,6 @@ raw2trace_t::process_offline_entry(raw2trace_thread_data_t *tdata,
                                    OUT bool *end_of_record, OUT bool *last_bb_handled,
                                    OUT bool *flush_decode_cache)
```

Comment:
I think some tests are failing b/c of too-long >90 lines -- please fix to get the tests green.